src/neo-asm-cli/test.asml.txt

```diff
@@ -0,0 +1,22 @@
+//comment 1 1 23
+//让我们荡起双桨
```

Comment:
In english is better for the community

Reply:
this sentence is so interesting ... BTW, I think this is used for testing the comment in ASM, so combine Chinese/English is ok.

Reply:
If is the same is ok for me xD